pages/testimonials.md

```diff
@@ -1,15 +1,28 @@
 ---
-title: Testimionials & Advice from Seniors
+layout: two-column
 permalink: "/testimonials"
-layout: page
----
+title: "Testimionials & Advice from Seniors"
+prologue: <div>Here are some testimonials and advice from seniors.</div>
```

Comment:
This seems to be a custom variable in the [front matter](https://jekyllrb.com/docs/front-matter/) of the page. What does it do? If you're not sure, please remove it.

Reply:
i created it as like an introduction to the 2 column content

Reply:
Ah, I see; that helps to separate it from the content.